brave/brave-core
# Tweak portfolio UI and add option to show hidden NFTs

## Resolves https://github.com/brave/brave-browser/issues/32545

![Screenshot_1693913055](https://github.com/brave/brave-core/assets/3308503/8233ec66-1b2e-4c0a-8f1a-6c0dea52261e)



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
- Open Brave Wallet and navigate to "NFTs" section by tapping on the tab at the top
- Observe NFTs and Hidden sections are shown
- Add a new NFT
- Observe the NFT is shown in the NFTs section
- Tap on "Edit Visible NFTs"
- Deselect the NFT and tap on "Save account watchlist"
- Observe the NFT is shown in the Hidden section
